CONTRIBUTING.md

```
@@ -80,6 +81,27 @@ git checkout -b type/краткое-описание
     \label{fig:acf-pacf}
 \end{figure}
 ```
+
+**Для Смысловых блоков**:
+- В тексте будут особым цветом выделяться следующие блоки:
```

markdownlint-cli2 (0.18.1)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)